articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: What is quantum computing?
+description: 
+author: natke
+ms.author:  
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.whatis
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science. Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+The basic unit of quantum computing is the `qubit`. Unlike a `bit` in classical computing, which can be `0` or `1`, a qubit can be in an infinite number of states - a **superposition** of both 0 and 1.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
+
+Qubits can also be **entangled**: when we make a measurement of one entangled qubit, it affects the state of the other, even if the qubits are physically far apart.
+
+## What can a quantum computer do?
+
+Quantum algorithms can be designed to take advantage of qubit superposition and entanglement to speed up classical algorithms, or to provide entirely new ways of modeling physical systems.
+
+One of the most famous quantum algorithms is `Shor's algorithm` for factorization, which makes the classically intractable problem of factorization of a large number into two prime numbers fast enough to challenge traditional cryptography.
```

Comment:
Similarly here, do we want "Shor's algorithm" to be typeset as a code literal? It may also be helpful to link to the relevant sample for integer factorization.

Reply:
I don't know if we should use this space to explain what is already explained in the section "What can we do with quantum computers".  Maybe we should save this space to explain "why quantum computers work" or a bit of the circuit model.